specification/purview/data-plane/Azure.Analytics.Purview.DataMap/preview/2024-03-01-preview/purviewdatamap.json

```diff
@@ -0,0 +1,8489 @@
+      "type": "object",
+      "description": "The value item of the search suggest.",
+      "properties": {
+        "@search.score": {
+          "type": "number",
+          "format": "float",
+          "description": "The search score calculated by the search engine. The results are ordered by\nsearch score by default.",
+          "x-ms-client-name": "searchScore"
+        },
+        "@search.text": {
+          "type": "string",
+          "description": "The target text that contains the keyword as prefix. The keyword is wrapped\nwith emphasis mark.",
+          "x-ms-client-name": "searchText"
+        },
+        "objectType": {
+          "type": "string",
+          "description": "The object type of the record. Object type is the top-level property to\ndistinguish whether a record is an asset or a term."
+        },
+        "createTime": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The create time of the record. The Unix epoch format."
+        },
+        "updateTime": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The last update time of the record. The Unix epoch format."
+        },
+        "id": {
+          "type": "string",
+          "description": "The GUID of the record."
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the record."
+        },
+        "qualifiedName": {
+          "type": "string",
+          "description": "The qualified name of the record."
+        },
+        "entityType": {
+          "type": "string",
+          "description": "The type name of the asset."
+        },
+        "description": {
+          "type": "string",
+          "description": "The description of the asset."
+        },
+        "endorsement": {
+          "type": "string",
+          "description": "The endorsement of the asset."
+        },
+        "owner": {
+          "type": "string",
+          "description": "The owner of the record."
+        },
+        "classification": {
+          "type": "array",
+          "description": "The classifications of the record.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "label": {
+          "type": "array",
+          "description": "The labels of the asset.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "term": {
+          "type": "array",
+          "description": "The terms assigned to the asset.",
+          "items": {
+            "$ref": "#/definitions/TermSearchResultValue"
+          },
+          "x-ms-identifiers": []
+        },
+        "contact": {
+          "type": "array",
+          "description": "The contacts of the asset.",
+          "items": {
+            "$ref": "#/definitions/ContactSearchResultValue"
+          }
+        },
+        "assetType": {
+          "type": "array",
+          "description": "The asset types of the asset
```

Comment:
Model with only `additionalProperties` should be used with caution.

Is there nothing you have on its schema?